python/cudf_polars/cudf_polars/dsl/translate.py

```diff
@@ -393,6 +404,9 @@ def translate_ir(visitor: NodeTraverser, *, n: int | None = None) -> ir.IR:
             f"No support for polars IR {version=}"
         )  # pragma: no cover; no such version for now.
 
+    IR._config = config
+    Expr._config = config
+
```

Comment:
Please no! Can we figure out a good way to pass the configuration into `evaluate`, which is where it is needed, rather than setting class attributes.